src/lib/core/_core.scss

```diff
@@ -41,4 +41,11 @@
   .mat-theme-loaded-marker {
     display: none;
   }
+
+  // Used when disabling global scrolling.
+  .cdk-global-scrollblock {
+    position: fixed;
+    overflow-y: scroll;
```

Comment:
It will, but we don't activate the strategy unless there was a scrollbar in the first place.